lib/landing_page.dart

```diff
@@ -0,0 +1,449 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+import 'dart:math';
+import 'package:url_launcher/url_launcher.dart';
+
+class LandingPage extends StatelessWidget {
```

Comment:
LandingPage should be split into components and comments should be added to increase readability.

Reply:
I agree. This will take me some time tho.
Why not keeping it that way until we decide if corona jump has a future or not? Otherwise its unused work. Ill add comments

Reply:
ok